### DISCUSSION TITLE
a few things (flower variants, better fences, orange birch leaves)

### DISCUSSION THREAD

**Discussion by freezeMeister** [BedrockTweaks/Files#366] [Ideas]
there are a few things i think would be cool in the pack like an option for variants in flowers, the fences that look like stripped wood (from vanilla tweaks) or even **orange** birch leaves (orange leaves not liver disease light brown many packs do)

**Comment by SquatchHunter** [comment]
community packs will be done after addons are
we still dont have a timeline
